go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/k0sproject/bootloose
 
-go 1.22.0
+go 1.23.0
```

Comment:
```suggestion
go 1.23
```

Reply:
The version is now not pinned so binaries will be magically built with whatever 1.23 the CI has.

Maybe the version should be pinned in the workflow?